qiita_db/support_files/patches/python_patches/40.py

```diff
@@ -0,0 +1,22 @@
+from random import SystemRandom
+from string import ascii_letters, digits
+
+from qiita_db.sql_connection import TRN
+
+pool = ascii_letters + digits
+with TRN:
```

Comment:
Perhaps out of the scope of this PR and more cause this is how we are doing the previous ones. However, this means that every DB clean for the test environment we will get new ids and we will need to reset the configuration files, which is not convenient. Why not just define some for each plugin?